client/js/table-view.js

```diff
@@ -82,11 +70,29 @@ class TableView {
 		.forEach(th => this.headerRowEl.appendChild(th));
 	}
 
+	renderTableFooter(){
+		removeChildren(this.footerRowEl);
+		for(var column=0; column <this.model.numCols; column++){
+			let sum =0; 
```

Comment:
Spacing issues. There should be a space after each operator (=, &&, ||, etc.) and complete expression.